stardog/http/client.py

```diff
@@ -45,6 +45,10 @@ def __init__(
             )
         self.session.auth = auth
 
+        healthcheck = self.url + "/admin/healthcheck"
```

Comment:
ditch the `self.url` from here, since `self.get()` is already calling `self.url`. keeping it will duplicate the endpointname in the request. or I guess you could just do:
`self.get("/admin/healthcheck")`